venom/message.py

```diff
@@ -74,6 +74,10 @@ def get(self, key, default=None):
             return default
 
     def __getitem__(self, key):
+        if key not in self._values:
```

Comment:
This is not good. You can instead update the setter code for field descriptors so that it sets based on the field name. __getitem__ and __setitem__ should just be simple getters and setters.